app/scripts/modules/core/src/presentation/forms/SpinFormik.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { Formik, FormikConfig } from 'formik';
+
+/**
+ * This component wraps the <Formik/> component, applying fixes and spinnaker opinions
+ * Use this component like you would use the <Formik/> component
+ */
+function SpinFormikImpl<Values extends {}>(props: FormikConfig<Values>, ref?: React.MutableRefObject<Formik<Values>>) {
+  const formikRef = ref || React.useRef<Formik<Values>>();
+  const defaultIsInitialValid = () => formikRef.current && Object.keys(formikRef.current.state.errors).length === 0;
+
+  // Run initial validation when the form is first rendered
+  React.useEffect(() => {
+    formikRef.current && formikRef.current.getFormikActions().validateForm();
+  }, [formikRef.current]);
+
+  return <Formik<Values> ref={formikRef} isInitialValid={props.isInitialValid || defaultIsInitialValid} {...props} />;
```

Comment:
Would there ever be cases where someone would want to opt out of this initial validation? And if so, would `isInitialValid={true}` be the way to do that?

Reply:
1) I don't know
2) I don't know

Reply:
I believe that formik v2 will deprecate `isInitialValid` paradigm but leave in backwards compat code